mmcv/runner/hooks/lr_updater.py

```diff
@@ -220,6 +220,66 @@ def get_lr(self, runner, base_lr):
         return annealing_cos(base_lr, target_lr, progress / max_progress)
 
 
+@HOOKS.register_module()
+class CosineRestartLrUpdaterHook(LrUpdaterHook):
+    """Cosine annealing with restarts learning rate scheme.
+
+    Args:
+        period (list[int]): Period for each cosine anneling cycle.
+        restart_weights (list[float]): Restart weights at each restart
+            iteration. Default: [1].
+        eta_min (float): The mimimum lr. Default: 0.
```

Comment:
1. May use min_lr and min_lr_ratio
2. May support `min_lr_ratio` to support relative LR schedule rather than absolute LR.

Reply:
Updated according to the comment.